### ISSUE TITLE
Lair [BCUS98112]  Game freezes on the second level

### ISSUE THREAD

**Issue by Nuke10** [RPCS3/rpcs3#13555]
### Quick summary

Roughly around the area i attack the boat the game freezes

### Attach a log file

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/10992406/RPCS3.log.gz)

### Attach capture files for visual issues

https://youtu.be/1t3dCLCKZv4

### System configuration

AMD Ryzen 9 5950X 16 Cores Processor,

Nvidia Geforce RTX 3800

RAM:64 Bit:

OS:windows 10 Home edition 64 Bit

**Comment by digant73** [comment]
is that issue still present?

**Comment by PmNz8** [comment]
Yes, I just tried it today and on version BCES00004 the game crashes during attack on the catapult ships.
